Northstar.Client/mod/scripts/vscripts/ui/menu_ns_serverbrowser.nut

```diff
@@ -1085,9 +1085,11 @@ void function ThreadedAuthAndConnectToServer( string password = "" )
 	}
 	else
 	{
+		var reason = NSGetAuthFailReason()
```

Comment:
```suggestion
		string reason = NSGetAuthFailReason()
```

Reply:
Lol, I missed that (it was there in the other one to begin with); I'm silly xD.